packages/shared-ux/link/redirect_app/impl/src/redirect_app_links.styles.ts

```diff
@@ -9,6 +9,9 @@
 import { css } from '@emotion/react';
 
 export const redirectAppLinksStyles = css({
-  display: 'flex',
+  display: 'inherit',
```

Comment:
This change makes the redirect app link inherit the display property of it's parent element, ideally this component should not have a presentational effect because it only handles links but for historical reasons and how markup has been built around this component we'd have to support this. 
The flex properties added are necessary to support existing markup that also expects that the `.KbnAppWrapper` class would be present on the `RedirectAppLinks` component, given that this component historically also accepted  classNames.